### PR TITLE
[CN-532] Add CronHotBackup documentation

### PR DESCRIPTION
**Pull request by mtyazici**

### CODE REVIEW

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 1]
```diff
@@ -112,7 +112,7 @@ For the full list of the supported expression, please check the link:https://pkg
 +
 [source,bash]
 ----
-kubectl apply -f ./hot-backup-scheduled.yaml
+kubectl apply -f ./cron-hot-backup.yaml
 ----
 
 === Checking the Status of a Backup
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/backup-restore.adoc:118]:
```suggestion
=== Checking the Status of a Local Backup
```
Also, should we say.... "To check the status of a local backup, run the following command:". Also do we need to provide an example to show how to check the status of a local scheduled backup/is it the same? Can you check the status of a specific scheduled backup, i.e. by filename?

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 2]
```diff
@@ -300,18 +300,18 @@ kubectl create secret generic <secret-name> \
...
 +
 [source,yaml,subs="attributes+"]
 ----
-include::ROOT:example$/hot-backup-agent-scheduled.yaml[]
+include::ROOT:example$/cron-hot-backup-agent.yaml[]
 ----
 +
-This field takes a valid cron expression. For example:
+`.spec.schedule` field takes a valid cron expression. For example:
 +
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/backup-restore.adoc:315]: For the table below, just some suggestions for slight rewordings.

At 10.30 AM every day

On 1st, 15th, and 25th of each month at midnight

On the first day of each month at midnight

For a full list of support expressions, see the library documentation.

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 3]
```diff
@@ -332,7 +332,7 @@ For the full list of the supported expression, please check the link:https://pkg
...
 [source,bash]
 ----
-kubectl apply -f ./hot-backup-agent-scheduled.yaml
+kubectl apply -f ./cron-hot-backup-agent.yaml
 ----
 
 === Checking the Status of a Backup
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/backup-restore.adoc:338]:
```suggestion
=== Checking the Status of an External Backup
```
Can you check the status of a specific scheduled backup, i.e. by filename?

Also, could we say: "To check the status of an external backup, run the following command:"

**Review by angelasimms — CHANGES_REQUESTED** [review]
Just a few suggestions. I couldn't see a preview of the up-to-date API reference so just based my review on the content of these files.

File: docs/modules/ROOT/examples/cron-hot-backup.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: CronHotBackup
+metadata:
+  name: cronhot-backup
```
[the remark this answers is not in the document]
Comment (mtyazici) [reply docs/modules/ROOT/examples/cron-hot-backup.yaml:4]: Sure, I will change it

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 5]
```diff
@@ -80,18 +80,18 @@ include::ROOT:example$/hot-backup.yaml[]
...
 +
 [source,yaml,subs="attributes+"]
 ----
-include::ROOT:example$/hot-backup-scheduled.yaml[]
+include::ROOT:example$/cron-hot-backup.yaml[]
 ----
 +
-This field takes a valid cron expression. For example:
+`.spec.schedule` field takes a valid cron expression. For example:
 +
```
[the remark this answers is not in the document]
Comment (mtyazici) [reply docs/modules/ROOT/pages/backup-restore.adoc:95]: Sure, I am updating it

File: docs/modules/ROOT/pages/backup-restore.adoc (hunk 2 above)
Comment (mtyazici) [reply docs/modules/ROOT/pages/backup-restore.adoc:315]: I am updating it

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 6]
```diff
@@ -80,42 +80,42 @@ include::ROOT:example$/hot-backup.yaml[]
...
 [source,bash]
 ----
-kubectl apply -f ./hot-backup-scheduled.yaml
+kubectl apply -f ./cron-hot-backup.yaml
 ----
 
-=== Checking the Status of a Backup
+=== Checking the Status of a Local Backup
 
 To check the status of a backup, run the following command:
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/backup-restore.adoc:120]:
```suggestion
To check the status of a local backup, run the following command:
```

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 7]
```diff
@@ -302,40 +302,40 @@ kubectl apply -f ./hot-backup-agent.yaml
...
 [source,bash]
 ----
-kubectl apply -f ./hot-backup-agent-scheduled.yaml
+kubectl apply -f ./cron-hot-backup-agent.yaml
 ----
 
-=== Checking the Status of a Backup
+=== Checking the Status of an External Backup
 
 To check the status of a backup, run the following command:
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/backup-restore.adoc:340]:
```suggestion
To check the status of an external backup, run the following command:
```

**Review by angelasimms — CHANGES_REQUESTED** [review]
Sorry, just a few minor updates. Apologies, I made a couple of errors.

**Review by angelasimms — APPROVED** [review]

**Review by JakeSCahill — APPROVED** [review]
LGTM, thanks guys.

**Pull request merged**